dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatHistory.cs

```diff
@@ -201,6 +202,13 @@ public void RemoveRange(int index, int count)
         this._messages.RemoveRange(index, count);
     }
 
+    /// <summary>
+    /// Returns true if the chat history contains a message with the specified role.
+    /// </summary>
+    /// <param name="role">Role of the message.</param>
+    /// <returns>true if the chat history contains a message with the specified role and otherwise false.</returns>
+    public bool Contains(AuthorRole role) => this._messages.Any(m => m.Role == role);
```

Comment:
Is this really common enough to need a dedicated method? Can't the one call site so the exact same thing?